keep-ui/app/alerts/alert-note-modal.tsx

```diff
@@ -0,0 +1,121 @@
+'use client';
+
+import React, { useContext, useState } from 'react';
+const ReactQuill = typeof window === 'object' ? require('react-quill') : () => false;
```

Comment:
If a component is a child of a client component, it'll be automatically turned into a client component

Reply:
I had an exception ("document is not defined" or something similar") and I found a solution here - https://github.com/zenoamaro/react-quill/issues/292

I'll add the issue as a comment since I'm not sure how to solve it otherwise